fiftyone/utils/kitti.py

```diff
@@ -1033,6 +1037,29 @@ def _proj_3d_to_right_camera(detections3d, calib, frame_size):
     return detections2d
 
 
+def _convert_point_lidar_to_camera(xyz, V2C, R0):
+    x, y, z = xyz
+    p = np.array([x, y, z, 1])
+    return tuple((R0 @ V2C @ p)[0:3])
+
+
+def convert_box_lidar_to_camera(boxes, V2C, R0):
```

Comment:
Is this method used in a blog post? It's not used in this PR.

The reason I bring this up is that I prefer that all public functions have full docstrings, but this one feels out of place because the rest of the module doesn't provide any public function to access these required inputs.

fwiw, this could be written as a single line in whatever place you're using this operation:

```py
# Convert point LIDAR to camera coordinates
x, y, z = (R0 @ V2C @ np.array([x, y, z, 1]))[0:3]
```